### PR TITLE
services/regulated-assets-approval-server-demo: bootstrap service for the regulated assets' approval server demo

### PR DESCRIPTION
**Pull request by marcelosalloum**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] ~This PR adds tests for the most critical parts of the new functionality or fixes.~
* [x] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [x] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Bootstrap a go service that will be used to serve a demo implementation of a [SEP-8] approval server.

### Why

After this PR is merged we can start implementing the Approval Server features.

### Known limitations

N/A

[SEP-8]: https://github.com/stellar/stellar-protocol/blob/7c795bb9abc606cd1e34764c4ba07900d58fe26e/ecosystem/sep-0008.md

### CODE REVIEW

File: services/regulated-assets-approval-server-demo/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+All notable changes to this project will be documented in this file. This
+project adheres to [Semantic Versioning](http://semver.org/).
+
+As this project is pre 1.0, breaking changes may happen for minor version bumps.
+A breaking change will get clearly notified in this log.
+
+## [v0.0.1] - 2021-03-09
+
+Initial release bootstraping an API service that doesn't contain any logic yet.
```
Comment (leighmcculloch) [inline services/regulated-assets-approval-server-demo/CHANGELOG.md:11]: ❓ Are you releasing today? This looks like the first commit. I think you can start with this project being unreleased. You're not releasing it, and you don't need to update this file on every change.
```suggestion
## Unreleased

Initial release.
```

File: services/regulated-assets-approval-server-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# regulated-assets-approval-server-demo
+
+```
+Varsion: v0.0.1
+Status: not functional
+```
+
+This is a [SEP-8] Approval Server **demo** based on SEP-8 v1.6.1. It is being
```
Comment (leighmcculloch) [inline services/regulated-assets-approval-server-demo/README.md:8]: 💭  Will this demo become our reference implementation? I think you could call it that if so. A reference implementation does not need to be appropriate for production, and so additionally calling out that it is intended for testing only would be good. It is implied with demo, but I don't see that term get used a whole lot in tech communities. Specifying that something is an emulator, or is only for testing is more common.
```suggestion
This is a [SEP-8] Approval Server reference implementation based on SEP-8 v1.6.1 intended for **testing only**. It is being
```

**Review by leighmcculloch — APPROVED** [review]
A question (❓) and thought (💭), but otherwise LGTM.

**Review by acharb — APPROVED** [review]
Nice 👍 , excited to see what this project looks like at the end. I haven't added a new project to this monorepo or used cobra yet but from what I read LGTM,  just had 1 comment

File: services/regulated-assets-approval-server-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Be used as an example of how regulated assets transactions can be validated
+   and revised by an anchor.
+2. serve as a demo server where wallets can test and validate their SEP-8
+   implementation.
+
+Please be aware this is a demo and its code is not mean to be used in a
+production application.
+
+## Usage
```
[the remark this answers is not in the document]
Comment (marcelosalloum) [reply services/regulated-assets-approval-server-demo/README.md:19]: Makes sense 👍  but I think it is implied in Go projects?

[the remark this answers is not in the document]
Comment (marcelosalloum) [reply services/regulated-assets-approval-server-demo/README.md:19]: Adding it in https://github.com/stellar/go/pull/3457/commits/d7c007e09ce94845cc01688ea366799008b03030

File: services/regulated-assets-approval-server-demo/CHANGELOG.md (hunk 1 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server-demo/CHANGELOG.md]: That sounds much better, thanks.
https://github.com/stellar/go/pull/3457/commits/8b5412bead2a61309c80687f76e78710ad869f95

File: services/regulated-assets-approval-server-demo/README.md (hunk 2 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server-demo/README.md]: Good point, I agree "demo" is probably not a standard word for this.
https://github.com/stellar/go/pull/3457/commits/91eed1be36008cf57b7623066ac52475286ed3d9

**Pull request merged**